wordpress-mobile/WordPress-iOS
# Jetpack Focus: Correctly handle large text within the fullscreen overlay

Fixes #19649

## Description
This PR fixes an issue where the fullscreen overlay text wasn't reacting to the system font size changes.

## Testing Instructions

1. Open the app
2. Make sure the "Jetpack Features Removal Phase one" flag is enabled from the debug menu
3. Navigate to the Sites list
4. Tap plus button
5. Tap "Create WordPress.com site"
6. The overlay is displayed
7. Switch to Settings > Accessibility > Display & Text Size > Larger Text
8. Increase Text Size
9. Switch back to the WordPress app
10. Make sure the overlay's content size increases.


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
